src/scheduler/pbs_sched.c

```diff
@@ -118,6 +118,7 @@ struct		connect_handle connection[NCONNECTS];
 int		connector;
 int		server_sock;
 int		second_connection = -1;
+static int	update_svr = 1;
```

Comment:
There is no need to make this global.  Make it a local variable in main().

Reply:
Moved this variable under main() function.